### DISCUSSION TITLE
implement AuthorizationController

### DISCUSSION THREAD

**Discussion by Angel0926** [quarkusio/quarkus#37607] [Q&A]

**Comment by sberyozkin** [comment]
@Angel0926 Sorry for a delay,  I do have a feeling that I have already tested it and indeed it was discussed before,  have you solved it ?

**Reply by Angel0926** [reply]
thank you ) yes i changed Priority to @Priority(Interceptor.Priority.LIBRARY_BEFORE) and that is worked
